### PR TITLE
KVM tenant cluster release 14.0.0 with Kubernetes 1.19

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/roadmap/issues/247

### CODE REVIEW

**Comment by yulianedyalkova** [comment]
CNCF tests passed:
```
○ → sonobuoy results sonobuoy-results.tar
Plugin: e2e
Status: passed
Total: 5484
Passed: 305
Failed: 0
Skipped: 5179

Plugin: systemd-logs
Status: passed
Total: 4
Passed: 4
Failed: 0
Skipped: 0
```

The task is failing on cleanup (https://gigantic.slack.com/archives/CHAHY6VMF/p1618917282076500).

**Comment by tfussell** [comment]
CNCF has now passed several times. I will run it one last time this evening to see if I can catch what caused the failure on the previous run. Manual cluster creation and upgrades are working well. I think this is just about ready to go.

**Review by cornelius-keller — APPROVED** [review]
looks good, lgtm

**Review by pipo02mix — APPROVED** [review]

File: kvm/v14.0.0/release.diff [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Generated with:                                               <
+# devctl release create --name 13.1.0 --provider kvm --base 13. <
```
Comment (stone-z) [inline kvm/v14.0.0/release.diff:2]: Is this truncated or is `--base 13.` just a neat shortcut?

**Review by stone-z — APPROVED** [review]
Looks good to me if CNCF has passed in this configuration (status shows failed in GitHub)

File: kvm/v14.0.0/release.diff (hunk 1 above)
Comment (tfussell) [reply kvm/v14.0.0/release.diff:2]: Unfortunately not any shortcut. `diff` truncates long lines.

**Comment by tfussell** [comment]
CNCF failed due to the default master size being too small on our test installation. I have a PR for cluster-service to increase the size after which this should pass consistently.

**Pull request merged**